.travis.yml

```diff
@@ -2,32 +2,70 @@ language: python
 sudo: false
 python:
   - "2.7"
+
+cache:
+  directories:
+    - node_modules
+    - $HOME/.cache/pip/
+
+matrix:
+  fast_finish: true
+
 env:
-  - TOX_ENV=flake8
-  - TOX_ENV=docs
-  - TOX_ENV=assets
-  - TOX_ENV=es
-  - TOX_ENV=addons-devhub-editors
-  - TOX_ENV=main
+  - TOX_ENV=flake8 APP=xx
+  - TOX_ENV=docs APP=xx
+  - TOX_ENV=assets APP=xx
+  - TOX_ENV=elasticsearch APP=xx
+  - TOX_ENV=main APP=abuse
+  - TOX_ENV=main APP=accounts
+  - TOX_ENV=main APP=addons
+  - TOX_ENV=main APP=amo
+  - TOX_ENV=main APP=api
+  - TOX_ENV=main APP=bandwagon
+  - TOX_ENV=main APP=blocklist
+  - TOX_ENV=main APP=devhub
+  - TOX_ENV=main APP=discovery
+  - TOX_ENV=main APP=editors
+  - TOX_ENV=main APP=files
+  - TOX_ENV=main APP=landfill
+  - TOX_ENV=main APP=lib
+  - TOX_ENV=main APP=paypal
+  - TOX_ENV=main APP=reviews
+  - TOX_ENV=main APP=search
+  - TOX_ENV=main APP=signing
+  - TOX_ENV=main APP=stats
+  - TOX_ENV=main APP=tags
+  - TOX_ENV=main APP=translations
+  - TOX_ENV=main APP=users
+  - TOX_ENV=main APP=zadmin
```

Comment:
I feel like we need a way to automate this, otherwise we **will** forget new apps.